lib/CapabilityDelegation.js

```diff
@@ -146,6 +157,15 @@ module.exports = class CapabilityDelegation extends ControllerProofPurpose {
           'or its controller.');
       }
 
+      // make sure invocationTarget is valid
+      const target = utils.getTarget(document);
+      const parentTarget = utils.getTarget(verifiedParentCapability);
+      if(!(parentTarget === target || (allowHierarchicalDelegation &&
+        target.startsWith(`${parentTarget}/`)))) {
+        throw new Error('The `invocationTarget` in a delegated capability ' +
```

Comment:
Minor observation.  In this PR and validate-allowed-action PR the keyword quoting is mixed.  Might be better to just standardize on double quotes, since those would work inside single quotes or template literals without escaping.